server/middleware/auth.js

```diff
@@ -7,12 +7,9 @@ const auth = (req, res, next) => {
   if (!token) return res.status(401).send("Not Authorized");
   const secret = process.env.JWT_KEY || "secret";
   try {
-    // verify token - user id and email
-    const { id, email } = jwt.verify(token, secret);
-    req.user = {
-      id,
-      email,
-    };
+    // verify token - user
+    const user = jwt.verify(token, secret);
+    req.user = user;
```

Comment:
We need to fetch the user here - so it would be something like:
req.user = await User.findById(user.id)

Reply:
Meaning we would only need to dispatch `FETCH_USER` if we needed up to date information on the user?